kohese-portal/common/models/KoheseModel.json

```diff
@@ -25,6 +25,10 @@
         "foreignKey": "id"
       }
     },
+    "repositoryId": {
```

Comment:
Is this a really a derived reference to the Repository?  It looks like the value being stored in this attribute is a reference.